README.md

```diff
@@ -106,25 +123,24 @@ in folder ``TESTDUMP``.
 python tests/main.py DUMP
 ```
 
-It requires *onnxruntime*, *numpy* for most of the models,
-*pandas* for transform related to text features,
+It requires *onnxruntime*, *numpy* for most models,
+*pandas* for transforms related to text features, and
 *scipy* for sparse features. One test also requires
 *keras* to test a custom operator. That means
 *sklearn* or any machine learning library is requested.
 
 ## Add a new converter
 
 Once the converter is implemented, a unit test is added
-to test it works. At the end of the unit test, function
+to confirm that it works. At the end of the unit test, function
 *dump_data_and_model* or any equivalent function must be called
 to dump the expected output and the converted model.
 Once these file are generated, a corresponding test must
 be added in *tests_backend* to compute the prediction
 with the runtime.
 
-
 # License
 [MIT License](LICENSE)
 
 ## Acknowledgments
-The package was developed by the following engineers and data scientists at Microsoft starting from winter 2017: Zeeshan Ahmed, Wei-Sheng Chin, Aidan Crook, Xavier Dupre, Costin Eseanu, Tom Finley, Lixin Gong, Scott Inglis, Pei Jiang, Ivan Matantsev, Prabhat Roy, M. Zeeshan Siddiqui, Shouheng Yi, Shauheen Zahirazami, Yiwen Zhu, Du Li, Xuan Li, Wenbing Li
+This package was developed by the following engineers and data scientists at Microsoft starting from winter 2017: Zeeshan Ahmed, Wei-Sheng Chin, Aidan Crook, Xavier Dupre, Costin Eseanu, Tom Finley, Lixin Gong, Scott Inglis, Pei Jiang, Ivan Matantsev, Prabhat Roy, M. Zeeshan Siddiqui, Shouheng Yi, Shauheen Zahirazami, Yiwen Zhu, Du Li, Xuan Li, Wenbing Li, Vinitra Swamy.
```

Comment:
Probably, we need remove the name list here. They are all the real names instead of the account ids.

Reply:
Removed acknowledgements section with latest update. :)

Reply:
@wenbingl Why are these names removed? and what is wrong with real names? These are the people (except last 4) that initially development this toolkit. We had listed their names here because they would never show up on github contributors since the repo was an internal VSTS repo when it was first developed, however, when it was open-sourced on Github for launch the VSTS history was wiped out. The folks that contributed to ONNXMLTools after the launch, their names will show up in github contributor list hence there is no need to list their names and expand this list.

The ethical thing to do would be to restore the original acknowledgment as below (which I can do):

_The initial version of this package was developed by the following engineers and data scientists at Microsoft during winter 2017: Zeeshan Ahmed, Wei-Sheng Chin, Aidan Crook, Xavier Dupre, Costin Eseanu, Tom Finley, Lixin Gong, Scott Inglis, Pei Jiang, Ivan Matantsev, Prabhat Roy, M. Zeeshan Siddiqui, Shouheng Yi, Shauheen Zahirazami, Yiwen Zhu._

Reply:
Putting Microsoft and Real names together in a public place is a typical information leakage, which is major concern. So either we can keep "engineers and data scientists at Microsoft " or a real name list if you'd like to

'Ethical' is a strong word here since from the contribution list, the most changes is indicated as the original team member, nothing related to this PR owner or reviewer.

Reply:
@wenbingl Let me seek legal guidance and get back to you.

Reply:
A guidance will be better but I don't know if you can find one for this specific case. Or if you can tell the name list here actually are agreed with all members, you can also submit a PR to fix it.